gatsby-node.js

```diff
@@ -12,6 +14,78 @@ const paginationPath = (page, totalPages, prefix = `/`) => {
   }
 }
 
+const getAlbumWithTitleId = curry((title, id) => {
+  return {
+    path: mangoSlugfy(title),
+    component: path.resolve(`./src/templates/album.js`),
+    context: {
+      albumId: id,
+      slugPath: mangoSlugfy(title),
+    },
+  }
+});
+
+
+const getPaginatedPageWithContentComponentPathPrefix = curry((content, component, pathPrefix, postsPerPage, tagKeyName, prefix) => {
```

Comment:
I don't know how are set your project definitions but I usually try to keep the line size under 80 characters mostly for legibility sake, so if you have something like this break down the parameters one per line would make it cleaner.